hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -241,6 +222,4 @@ def get_threat_exchange_reaction_labels(
 
     match_message = MatchMessage("key", "hash", [banked_signal])
 
-    action_label_to_action_rules = get_actions_to_take(match_message, action_rules)
-
-    print(action_label_to_action_rules)
+    action_label_to_action_rules = get_actions_to_take(match_message, action_rules)
```

Comment:
Looks like this isn't formatted with black. There should be an ending newline. Also, why nix the print statement? This was my debugging area.

Reply:
Oops. Thought I added the print. Will add back

Black isn't complaining for me